.pipelines/ci.yml

```diff
@@ -32,25 +36,41 @@ jobs:
     name: ARO-CI
   steps:
   - template: ./templates/template-checkout.yml
+
   - script: |
       set -xe
-      make test-go
```

Comment:
1. Does it really make sense to run separate steps? "Run Golang code generate" and "Validate Golang code" do not seem to produce a lot of noise.

2. Also we are no longer testing whether the code actually builds:  `make test-go` still includes `go build -tags containers_image_openpgp ./...`, but becuase now we run separate make targets in CI, we do not run it.

Reply:
1. You are right, there is not much noise. I have split the items to provide dead simple issue search. When the items are split, it is obvious where the error came from just from a glance at he output. When the items are not split, I need to open the logs and grep for `exit 1` and mentally attach it to the proper code block.
2. Yep, I have simply missed it. Now it is back and doing the job.